libs/console/full/src/cons_tty.c

```diff
@@ -28,19 +28,21 @@ int g_console_is_init;
 /** Indicates whether the previous line of output was completed. */
```

Comment:
Hi Fabio!

Thanks for the PR, generally looks good.

If you can please add this as conditionally compiled, it would be greatly appreciated.

Specifically, look to use a feature in newt, which if enabled, will allow console history to be present -- since this feature takes memory that we probably don't want to spend on lower end platforms like the NRF51.  

A good example of using a feature to make this conditionally compiled exists in the statistics package (sys/stats) -- grep for STATS_NAME_ENABLE.

Thank you for the PR!